openedx_tagging/core/tagging/api.py

```diff
@@ -10,13 +10,28 @@
 Please look at the models.py file for more information about the kinds of data
 are stored in this app.
 """
+import csv
+import json
+from enum import Enum
+from io import StringIO, BytesIO, TextIOWrapper
 from typing import List, Type
 
+from django.db import transaction
 from django.db.models import QuerySet
+from django.core.exceptions import ObjectDoesNotExist
 from django.utils.translation import gettext_lazy as _
 
 from .models import ObjectTag, Tag, Taxonomy
 
+csv_fields = ['id', 'name', 'parent_id', 'parent_name']
```

Comment:
nit: would like to make it clear that this is a constant, and it's not part of the externally-exportable python api:

```suggestion
_CSV_FIELDS = ['id', 'name', 'parent_id', 'parent_name']
```